corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -45,6 +45,24 @@ <h4>
       <i class="fa fa-warning"></i>
       {% trans "Note: Continuing will change your current subscription." %}
     </h4>
+  {% elif is_same_edition %}
+    <h4>
```

Comment:
move `h4` outside of conditional

Reply:
https://github.com/dimagi/commcare-hq/pull/36448/commits/19c0784bb9fe01b2bb6f3d5173777214f04d2a7f also dealt with trans vs blocktrans here